recipe/meta.yaml

```
@@ -16,7 +16,10 @@ build:
   # Skip s390x and ppc64 since they lack bazel.
   skip: True # [s390x]
   # skipping win until build pipeline for win works
-  skip: True # [win]
+  skip: True # [win or (osx and x86_64)]
+  # catboost is not available for py38 on osx-64 for an issue, see:
+  # https://github.com/AnacondaRecipes/catboost-feedstock/blob/7fdf7753e6d9da42340a5aeb5c29fc4451dd6d57/recipe/meta.yaml#L17
+  skip: True  # [(osx and x86_64) and py==38]
```

```suggestion
  skip: True # [win]
  # catboost is not available for py38 on osx-64 for an issue, see:
  # https://github.com/AnacondaRecipes/catboost-feedstock/blob/7fdf7753e6d9da42340a5aeb5c29fc4451dd6d57/recipe/meta.yaml#L17
  skip: True  # [(osx and x86_64) and py==38]
```
If only `catboost` **py38** isn't available on `osx-64`

---

Totally, 
thank you @skupr-anaconda! Fixed it